app/renderer/bookmarks.js

```diff
@@ -0,0 +1,512 @@
+        .includes(`${option}=`))) {
+        const eachOption = input.join("").split("~")
+        selectedBookmarks = storedBookmarkData.filter(b => {
+            let matchedOptions = 0
+            eachOption.forEach(e => {
+                const [key, value] = e.split("=")
+                if (key === "keywords") {
+                    const eachValue = value.split(",")
+                    let matchedValues = 0
+                    eachValue.forEach(v => {
+                        if (v.startsWith("!")) {
+                            if (!b[key].includes(v.substring(1))) {
+                                matchedValues += 1
+                            }
+                        } else if (b[key].includes(v)) {
+                            matchedValues += 1
+                        }
+                    })
+                    if (eachValue.length === matchedValues) {
+                        matchedOptions += 1
+                    }
+                }
+                if (b[key] === value) {
+                    matchedOptions += 1
+                }
+            })
+            if (matchedOptions === eachOption.length) {
+                return true
+            }
+            return false
+        })
+    } else {
+        const individualBookmark = storedBookmarkData.filter(
+            e => e.name.replace(specialChars, "") === input.join(" ")
+                .replace(specialChars, ""))
+        const bookmarksSelectedByKeyword = storedBookmarkData.filter(
+            e => e.keywords.find(k => k === input.join("")))
+        selectedBookmarks
+            = individualBookmark.concat(bookmarksSelectedByKeyword)
+    }
+    return selectedBookmarks
+}
+
+/**
+ * Filter folders based on input.
+ * @param {string[]} input - The input to parse.
+ * @returns {Folder | null | undefined}
+ */
+const matchFoldersToInput = input => {
+    /** @type {Folder | null | undefined} */
+    let selectedFolder = null
+    if (input[0] && validFoldersOptions.some(option => input[0]
+        .includes(`${option}=`))) {
+        const [key, value] = input[0].split("=")
+        if (key === "path") {
+            selectedFolder = bookmarkData.folders.find(f => f.path === value)
+        }
+    }
+    return selectedFolder
+}
+
+/**
+ * Load bookmark based on the input.
+ * @param {string[]} input - The input to parse.
+ */
+const loadBookmark = input => {
+    const selectedBookmarks = matchBookmarksToInput(input)
+    const {addTab, navigateTo} = require("./tabs")
+    if (selectedBookmarks.length === 0) {
+        notify({
+            "id": "bookmarks.notfound",
+            "src": "user",
+            "type": "dialog"
+        })
+    } else if (selectedBookmarks.length === 1) {
+        navigateTo("user", selectedBookmarks[0].url)
+    } else {
+        notify({
+            "fields": [String(selectedBookmarks.length)],
+            "id": "bookmarks.loaded",
+            "src": "user",
+            "type": "dialog"
```

Comment:
Same here, only show warnings not success messages for something that triggers a big visual change anyway, and use the "warning" type instead of "dialog".